src/components/StyledButton/StyledButton.tsx

```diff
@@ -0,0 +1,27 @@
+import styles from './styles';
```

Comment:
is this the same button that annie added to her auth styling PR? if not this may need to be renamed

Reply:
Yeah, want me to pull her new PR from main and make sure there aren't duplicates or conflicts? and then merge